packages/forklift-console-plugin/src/modules/Providers/utils/validators/provider/openshift/validateOpenshiftURL.ts

```diff
@@ -1,12 +1,19 @@
 import { validateURL, ValidationMsg } from '../../common';
 
 export const validateOpenshiftURL = (url: string | number): ValidationMsg => {
+  if (url === undefined) {
```

Comment:
nitpick: can you add a comment like in line 11, explaining why we check for `undefined` separately ?

Reply:
Sure, done for all places where we check for `undefined` separately.